Assets/HoloToolkit/Build/Editor/BuildDeployWindow.cs

```diff
@@ -224,32 +224,34 @@ private void OnGUI()
             }
             else
             {
-                var isLocatorSearching = XdeGuestLocator.IsSearching;
-                var doesLocatorHaveData = XdeGuestLocator.HasData;
+                var locatorIsSearching = XdeGuestLocator.IsSearching;
+                var locatorHasData = XdeGuestLocator.HasData;
                 var xdeGuestIpAddress = XdeGuestLocator.GuestIpAddress;
 
                 // Queue up a repaint if we're still busy, or we'll get stuck
                 // in a disabled state.
 
-                if (isLocatorSearching)
+                if (locatorIsSearching)
                     Repaint();
```

Comment:
General NIT: It's nice to have curly braces around the if's to future proof and be consistent.